docs/en/ingest-management/elastic-agent/advanced-kubernetes-managed-by-fleet.asciidoc

```diff
@@ -0,0 +1,101 @@
+[[advanced-kubernetes-managed-by-fleet]]
+= Advanced {agent} configuration managed by {fleet}
+
+For basic {agent} managed by {fleet} scenarios follow the steps in <<running-on-kubernetes-managed-by-fleet>>.
+
+On managed {agent} installations it can be useful to provide the ability to configure more advanced options, such as the configuration of providers during startup. Refer to <<providers>> for more details.
+
+Following steps demonstrate above scenario:
+
+[discrete]
+== Step 1: Download the {agent} manifest
+
+include::run-container-common/download-elastic-agent.asciidoc[]
+
+[discrete]
+== Step 2: Create a new configmap
+
+[source,yaml]
+.Create a new configmap
+------------------------------------------------
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: agent-node-datastreams
+  namespace: kube-system
+  labels:
+    k8s-app: elastic-agent
+data:
+  agent.yml: |-
+    providers.kubernetes_leaderelection.enabled: false
+    fleet.enabled: true
+    fleet.access_token: "<FLEET_ENROLLMENT_TOKEN>"
+---
+------------------------------------------------
+
+Notes:: 
+1. The above example demonstrates how to disable the `kubernetes_leaderelection` provider. For alternative scenarios you can follow the same procedure. For example, the following settings configure the `kubernetes` provider metadata enrichment:
+
+  [source,yaml]
+  .Example of configmap for kubernetes provider configuration
+  ------------------------------------------------
+  apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: agent-node-datastreams
+    namespace: kube-system
+    labels:
+      k8s-app: elastic-agent
+  data:
+    agent.yml: |-
+      providers.kubernetes: 
+        add_resource_metadata: 
+          deployment: true
+          cronjob: true
+      fleet.enabled: true
+      fleet.access_token: "<FLEET_ENROLLMENT_TOKEN>"
+  ---
+  ------------------------------------------------
+
+2. Find more information about https://www.elastic.co/guide/en/fleet/current/fleet-enrollment-tokens.html[Enrollment Tokens].
```

Comment:
```suggestion
Notes:: 
1. Find more information about https://www.elastic.co/guide/en/fleet/current/fleet-enrollment-tokens.html[Enrollment Tokens].
```

After I remove the spaces from the code example I got another error. This will fix that other error.